package.json

```diff
@@ -46,6 +46,10 @@
     "watchify": "^4.0.0",
     "web-ext": "^6.4.0"
   },
+  "browserslist": {
+    "chrome": "85",
+    "firefox": "68"
```

Comment:
We can bump the Firefox requirement to v75 at the same time.

Reply:
Done, thanks.